lib/checks/at_vsn.ex

```diff
@@ -0,0 +1,41 @@
+defmodule Archethic.Checks.AtVsn do
+  @moduledoc """
+  Check that GenServer & GenStateMachine have a @vsn attribute
+  """
+
+  use Credo.Check
```

Comment:
Actually when running credo is inform there is an check that return an issue, but credo does not handle it as an issue so credo pass without error.
To fix this we can add an category to the check like `:warning` or `:consistency` so credo will report it as an found issue
ie: `use Credo.Check, category: :warning`